x-pack/legacy/plugins/uptime/server/lib/adapters/monitor_states/elasticsearch_monitor_states_adapter.ts

```diff
@@ -24,25 +21,23 @@ export interface QueryContext {
   statusFilter?: string;
 }
 
-export class ElasticsearchMonitorStatesAdapter implements UMMonitorStatesAdapter {
-  constructor(private readonly database: DatabaseAdapter) {
-    this.database = database;
-  }
-
+export const elasticsearchMonitorStatesAdapter: UMMonitorStatesAdapter = {
   // Gets a page of monitor states.
-  public async getMonitorStates(
-    request: any,
-    dateRangeStart: string,
-    dateRangeEnd: string,
-    pagination: CursorPagination = CONTEXT_DEFAULTS.CURSOR_PAGINATION,
-    filters?: string | null,
-    statusFilter?: string
-  ): Promise<GetMonitorStatesResult> {
+  getMonitorStates: async ({
+    callES,
+    dateRangeStart,
+    dateRangeEnd,
+    pagination,
+    filters,
+    statusFilter,
```

Comment:
Probably not related to this PR, but i have always been confused by filters and statusFilter, i mean they always gives my brain a challenge to process the information. "Ok we have  filters and then we have statusFilter :) let's think about it , what they means."
i guess we should probably rename statusFilter to monitorStatus or maybe just merge it with filters.or maybe it's fine, it's just me :D

Reply:
We should chat about this, I'm realizing maybe @andrewvc and myself have never explained why we have this weirdness. I'd be fine to rename it to something else in a separate PR if we can think of a less confusing way to refer to it.